clang-tools-extra/clang-tidy/llvmlibc/NamespaceConstants.h

```diff
@@ -11,6 +11,8 @@
 namespace clang::tidy::llvm_libc {
 
 const static llvm::StringRef RequiredNamespaceStart = "__llvm_libc";
+const static llvm::StringRef RequiredNamespaceHiddenMacroName =
```

Comment:
I think it would be better to call this one "...NamespaceDecl" and the other one "...NamespaceRef".